test/expected/build/library.j/dest/resources/library/j/some.js

```diff
@@ -1,20 +1,103 @@
 /*!
  * ${copyright}
  */
+sap.ui.define([
+	"sap/ui/core/Control",
+	"./library",
+	"./library-core"
+], (Control, { MyValidEnum, ThisIsEnumToo: RenamedEnum }, coreLibrary) => {
 
-sap.ui.define(
-	["./dependency-es6-1"],
-	["./dependency-es6-2"],
-	["./dependency-es6-3"],
-	function (dep1, dep2, dep3) {
-		"use strict";
-
-		/**
-		 * @alias library.j
-		 * @namespace
-		 * @public
-		 */
-		var SomeFunction = function () {};
-	},
-	/* bExport= */ true
-);
+	const { AnotherValidEnum } = coreLibrary;
+	const { Buzz } = AnotherValidEnum;
+	const { AnotherValidEnum: AnotherRenamedEnum } = coreLibrary;
+	const { H1 } = sap.ui.core.TitleLevel;
+	const { Value2: RenamedValue2 } = RenamedEnum;
+
+	/**
+	 * @class
+	 * My super documentation of this class
+	 *
+	 * @extends sap.ui.core.Control
+	 *
+	 * @author SAP SE
+	 * @version ${version}
+	 *
+	 * @public
+	 * @alias library.j.ValidPropertyDefaultValue
+	 * @ui5-metamodel text
+	 */
+	var ValidPropertyDefaultValue = Control.extend(
+		"library.j.ValidPropertyDefaultValue",
```

Comment:
Control name doesn't fit to the name of the file / module (`library/j/ValidPropertyDefaultValue ` vs `library/j/some.js`).

Reply:
Done